mlir/lib/Dialect/Rock/IR/MfmaInsnGroup.cpp

```diff
@@ -237,10 +237,13 @@ auto getMfmaInsnGroupAttrMap = []() {
                       {1, 0, MFMAPermB::none},
                       {1, 1, MFMAPermB::none}}}},
                    {{MfmaTypeId::Fp16TyId, 64, 64},
-                    {ROCDL::mfma_f32_32x32x4f16::getOperationName(),
-                     1,
-                     1,
-                     {{1, 0, MFMAPermB::none}, {1, 1, MFMAPermB::none}}}},
+                    {ROCDL::mfma_f32_32x32x8f16::getOperationName(),
+                     2,
+                     2,
+                     {{0, 0, MFMAPermB::none},
```

Comment:
This is what I was missing! Thanks Jerry, can I ask you why this needs to be like this?

Reply:
This is because 64 x 64 tile is too large and has to split into 2 repeats in M dimension and 2 repeats in N dimension. For the total of 2 x 2 = 4 xdlops instructions, each need to come with a `{0, 0, none}` flag.